lib/topic_query.rb

```diff
@@ -365,6 +365,17 @@ def list_private_messages_tag(user)
     create_list(:private_messages, {}, list)
   end
 
+  def list_private_messages_warnings(user)
+    list = private_messages_for(user, :user)
+    list = list.where('NOT EXISTS (
```

Comment:
Perhaps add a comment explaining what this is? It seems to be "you have not replied to this topic" but a clarification would be a good idea.

Reply:
Wait yeah, that doesn't look like the correct query. I need to go back and figure out what this was supposed to do, because it's not correct.

Reply:
This is supposed to be `topics.user_id = ?` - official warnings that **you created**.

Reply:
Added a test to cover this.